xbmc/cores/VideoPlayer/VideoRenderers/HwDecRender/DRMPRIMEEGL.h

```diff
@@ -0,0 +1,50 @@
+/*
+ *      Copyright (C) 2007-2017 Team XBMC
+ *      http://xbmc.org
+ *
+ *  This Program is free software; you can redistribute it and/or modify
+ *  it under the terms of the GNU General Public License as published by
+ *  the Free Software Foundation; either version 2, or (at your option)
+ *  any later version.
+ *
+ *  This Program is distributed in the hope that it will be useful,
+ *  but WITHOUT ANY WARRANTY; without even the implied warranty of
+ *  MERCHANTABILITY or FITNESS FOR A PARTICULAR PURPOSE. See the
+ *  GNU General Public License for more details.
+ *
+ *  You should have received a copy of the GNU General Public License
+ *  along with XBMC; see the file COPYING.  If not, see
+ *  <http://www.gnu.org/licenses/>.
+ *
+ */
+
+#pragma once
+
+#include "cores/VideoPlayer/DVDCodecs/Video/DVDVideoCodecDRMPRIME.h"
+
+#define GL_GLEXT_PROTOTYPES
```

Comment:
Is it wise to set this in a header? What happens if `gl2ext.h` was already included?

Reply:
it's not needed now that we check the extension manually

Reply:
even better :+1: